include/config/config_safeguards.h

```diff
@@ -195,6 +195,11 @@
  * config_rom.h
  */
 
+#ifdef VERSION_SH
+    #undef ENABLE_RUMBLE
+    #define ENABLE_RUMBLE
+#endif // VERSION_SH
+
```

Comment:
I'd remove this tbh; not only do we not support SH anyway, but I don't see a reason why we would need to force rumble for it when we have the config option.

Reply:
Removed